py/pykms/pykmsutil.cpp

```diff
@@ -26,10 +26,13 @@ void init_pykmstest(py::module &m)
 			.def("reserve_connector", (Connector* (ResourceManager::*)(const string& name))&ResourceManager::reserve_connector,
 			     py::arg("name") = string())
 			.def("reserve_crtc", &ResourceManager::reserve_crtc)
-			.def("reserve_plane", &ResourceManager::reserve_plane,
+			.def("reserve_plane", (Plane* (ResourceManager::*)(Crtc*, Plane::Type, PixelFormat)) &ResourceManager::reserve_plane,
 			     py::arg("crtc"),
```

Comment:
Earlier when there was only one ResourceManager::reserve_plane(), there was no specify the parameter types and prototype is simply copied from method prototype. Now the casting is needed to differentiate between two overloaded prototypes. I'll add also a third prototype having unsigned type for typemask for convenience so that a result of operator| is accepted without a cast. It seems unsigned are not implicitly converted to regular enums if needed, but conversion from enum to unsigned is implicit.